cdap-source-control/pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-data-fabric</artifactId>
```

Comment:
is this needed? It looks like DefaultRepositoryStore is already in data-fabric?

Reply:
Yeah this is adding the `DefaultRepositoryStore` dependency in source-control module